include/walker-dropdown.php

```diff
@@ -34,11 +34,12 @@ class PLL_Walker_Dropdown extends PLL_Walker {
 	public function start_el( &$output, $element, $depth = 0, $args = array(), $current_object_id = 0 ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 		$value_type = $args['value'];
 		$output .= sprintf(
-			"\t" . '<option value="%1$s"%2$s%3$s>%4$s</option>' . "\n",
+			"\t" . '<option value="%1$s"%2$s%3$s data-pll-lang="%5$s">%4$s</option>' . "\n",
 			'url' === $value_type ? esc_url( $element->$value_type ) : esc_attr( $element->$value_type ),
 			! empty( $element->w3c ) ? sprintf( ' lang="%s"', esc_attr( $element->w3c ) ) : '',
 			selected( isset( $args['selected'] ) && $args['selected'] === $element->$value_type, true, false ),
-			esc_html( $element->name )
+			esc_html( $element->name ),
+			esc_html( wp_json_encode( $element ) )
```

Comment:
If `$element` is an instance of  `PLL_Language`, we should use the method `to_array()` and re-cast into object IMHO.

Reply:
As discussed, already done [here](https://github.com/polylang/polylang/blob/8b4d211152b3e5bd0037d1639d6537bc35303f5e/include/walker.php#L40).